content/en/docs/File Format/implementationstatus.md

```diff
@@ -45,64 +45,63 @@ Implementations:
-| PLAIN_DICTIONARY                          |       |       |       |       |  ✅   |
-| RLE_DICTIONARY                            |       |       |       |       |  ✅   |
-| RLE                                       |       |       |       |       |  ✅   |
-| BIT_PACKED (deprecated)                   |       |       |       |       |  (R)  |
-| DELTA_BINARY_PACKED                       |       |       |       |       |  ✅   |
-| DELTA_LENGTH_BYTE_ARRAY                   |       |       |       |       |  ✅   |
-| DELTA_BYTE_ARRAY                          |       |       |       |       |  ✅   |
-| BYTE_STREAM_SPLIT                         |       |       |       |       |  ✅   |
+| PLAIN                                     |  ✅   |  ✅   |       |       |  ✅   |
+| PLAIN_DICTIONARY                          |  ✅   |  ✅   |       |       |  ✅   |
+| RLE_DICTIONARY                            |  ✅   |  ✅   |       |       |  ✅   |
+| RLE                                       |  ✅   |  ✅   |       |       |  ✅   |
+| BIT_PACKED (deprecated)                   |  ✅   |  ✅   |       |       |  (R)  |
+| DELTA_BINARY_PACKED                       |  ✅   |  ✅   |       |       |  ✅   |
+| DELTA_LENGTH_BYTE_ARRAY                   |  ✅   |  ✅   |       |       |  ✅   |
+| DELTA_BYTE_ARRAY                          |  ✅   |  ✅   |       |       |  ✅   |
+| BYTE_STREAM_SPLIT                         |  ✅   |  ✅   |       |       |  ✅   |
 
 ### Compressions
 
 | Compression                               | C++   | Java  | Go    | Rust  | cuDF  |
 | ----------------------------------------- | ----- | ----- | ----- | ----- | ----- |
-| UNCOMPRESSED                              |       |       |       |       |  ✅   |
-| BROTLI                                    |       |       |       |       |  (R)  |
-| GZIP                                      |       |       |       |       |  (R)  |
-| LZ4 (deprecated)                          |       |       |       |       |  ❌   |
-| LZ4_RAW                                   |       |       |       |       |  ✅   |
-| LZO                                       |       |       |       |       |  ❌   |
-| SNAPPY                                    |       |       |       |       |  ✅   |
-| ZSTD                                      |       |       |       |       |  ✅   |
+| UNCOMPRESSED                              |  ✅   |  ✅   |       |       |  ✅   |
+| GZIP                                      |  ✅   |  ✅   |       |       |  (R)  |
+| LZ4 (deprecated)                          |  ✅   |  ✅   |       |       |  ❌   |
+| LZ4_RAW                                   |  ✅   |  ✅   |       |       |  ✅   |
+| LZO                                       |  ❌   |  ✅   |       |       |  ❌   |
+| SNAPPY                                    |  ✅   |  ✅   |       |       |  ✅   |
+| ZSTD                                      |  ✅   |  ✅   |       |       |  ✅   |
```

Comment:
Why have you removed `BROTLI`?

In the `parquet-java` tests we exclude `LZO` and `LZ4` because they are shipped with Hadoop. Can we say that these are supported?

Reply:
Oh sorry, I think BROTLI was deleted by accident.

> In the parquet-java tests we exclude LZO and LZ4 because they are shipped with Hadoop. Can we say that these are supported?

I have noticed that as well. I think the feature is supported but by default they are not enabled, right?

Reply:
Not sure how to represent this or if we want to, but `parquet-java` is capable of handling these compressions (read or write) only if the related hadoop codecs are available in the environment (so not shipped with the library).

Reply:
Let me mark them as unsupported since users cannot use it in the most common cases.